libs/permission-checker/src/StorageApiToken.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker;
+
+class StorageApiToken
```

Comment:
a potřebujem teda tuhle classu? já bych ty metody strčil do StorageApiTokenInterface, stejně je tam reálně máme

Reply:
No nemame, protoze nemame (a IMHO nechceme) mit mimo ten permission checker enumy na feature/role. Takle je to postaveny, aby psani tech ckeckeru bylo co nejprimocarejsi.

Reply:
No taky by klidně mohli bejt součástí AbstractCheck a ušetřili bysme si tuhle kašpařinu.

jakože `if (!$token->hasFeature(Feature::QUEUE_V2))` -> if (!$this->tokenHasFeature($token, Feature::QUEUE_V2))`

Reply:
Slo by.. me to jen prislo takle nejcitelnejsi.